From 294eb24385653395fc6188a182b222f1c56baf1a Mon Sep 17 00:00:00 2001
From: Yang Yingliang <yangyingliang@huawei.com>
Date: Fri, 26 Feb 2021 20:21:22 +0800
Subject: [PATCH] arm64/mpam: fix compile warning
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: 294eb24385653395fc6188a182b222f1c56baf1a

MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

hulk inclusion
category: bugfix
bugzilla: 48265
CVE: NA

--------------------------------

Fix the compile warning
fs/resctrlfs.c: In function ‘resctrl_mount’:
fs/resctrlfs.c:358:1: warning: label ‘out_mongrp’ defined but not used [-Wunused-label]
 out_mongrp:

Signed-off-by: Yang Yingliang <yangyingliang@huawei.com>
Signed-off-by: Wang ShaoBo <bobo.shaobowang@huawei.com>
Reviewed-by: Cheng Jian <cj.chengjian@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 fs/resctrlfs.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fs/resctrlfs.c b/fs/resctrlfs.c
index 31ecedf433ec..4377406be131 100644
--- a/fs/resctrlfs.c
+++ b/fs/resctrlfs.c
@@ -352,8 +352,8 @@ static int resctrl_get_tree(struct fs_context *fc)
 #ifndef CONFIG_ARM64 /* [FIXME] arch specific code */
 	if (resctrl_mon_capable)
 		kernfs_remove(kn_mondata);
-#endif
 out_mongrp:
+#endif
 	if (resctrl_mon_capable)
 		kernfs_remove(kn_mongrp);
 out_info:
-- 
2.26.2

